tuftool/src/root.rs

```diff
@@ -225,15 +225,23 @@ impl Command {
     }
 
     #[allow(clippy::borrowed_box)]
-    fn add_key(path: &Path, roles: &[RoleType], key_source: &Box<dyn KeySource>) -> Result<()> {
+    fn add_key(
+        path: &Path,
+        roles: &[RoleType],
+        key_source: &Vec<Box<dyn KeySource>>,
+    ) -> Result<()> {
         let mut root: Signed<Root> = load_file(path)?;
-        let key_pair = key_source
-            .as_sign()
-            .context(error::KeyPairFromKeySourceSnafu)?
-            .tuf_key();
-        let key_id = hex::encode(add_key(&mut root.signed, roles, key_pair)?);
         clear_sigs(&mut root);
-        println!("{key_id}");
+
+        for ks in key_source {
+            let key_pair = ks
+                .as_sign()
+                .context(error::KeyPairFromKeySourceSnafu)?
+                .tuf_key();
+            let key_id = hex::encode(add_key(&mut root.signed, roles, key_pair)?);
+            println!("{key_id}");
```

Comment:
Maybe a more descriptive log message would be helpful for what this output is.

Reply:
This actually is just bubbling up the [`KeyIdSnafu`](https://github.com/awslabs/tough/blob/develop/tuftool/src/error.rs#L165) or [`KeyDuplicateSnafu`](https://github.com/awslabs/tough/blob/develop/tuftool/src/error.rs#L159) errors from `add_key`: https://github.com/awslabs/tough/blob/develop/tuftool/src/root.rs#L423-L426

I don't think any more useful context can be added from here, but please do let me know if I'm missing anything.

Reply:
I was referring to the `println!("{key_id}");`. I think it would help contextualize what the hexstring being printed there is.

Reply:
Ah, makes sense. Quick update coming.